packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 
+import ApolloClient, {ApolloClientOptions} from 'apollo-client';
```

Comment:
Please add `apollo-client` to the `dependencies` list. It is sadly missing from before...><...